syntax = "proto3";

package GrpcSample;;

message AddReq {
	int32 lhs = 1;
	int32 rhs = 2;
}

message AddResp {
	int32 result = 1;
}

service Calculator{
	rpc Add(AddReq) returns (AddResp); 
}
